### PR TITLE
Slight PSS documentation

### PR DESCRIPTION
**Pull request by AsparagusEduardo**

### CODE REVIEW

File: src/pokemon_storage_system.c [hunk 1]
```diff
@@ -231,7 +231,7 @@ struct PokemonStorageSystemData
     struct Sprite *field_D94;
     struct Sprite *field_D98[2];
     u16 *field_DA0;
-    struct PokemonMarkMenu field_DA4;
+    struct PokemonMarkMenu monMarks;
```
Comment (huderlem) [inline src/pokemon_storage_system.c]: I think `markMenu` would be better than `monMarks`.

File: src/pokemon_storage_system.c [hunk 2]
```diff
@@ -2165,7 +2165,7 @@ static void Cb2_EnterPSS(u8 boxOption)
     {
         sPSSData->boxOption = boxOption;
         sPSSData->isReshowingPSS = FALSE;
-        gUnknown_02039D12 = 0;
+        sMovingItemId = 0;
```
Comment (huderlem) [inline src/pokemon_storage_system.c]: `ITEM_NONE`

File: src/pokemon_storage_system.c [hunk 3]
```diff
@@ -3863,9 +3863,9 @@ static void Cb_ChangeScreen(u8 taskId)
     u8 screenChangeType = sPSSData->screenChangeType;
 
     if (sPSSData->boxOption == BOX_OPTION_MOVE_ITEMS && IsActiveItemMoving() == TRUE)
-        gUnknown_02039D12 = GetMovingItem();
+        sMovingItemId = GetMovingItem();
     else
-        gUnknown_02039D12 = 0;
+        sMovingItemId = 0;
```
Comment (huderlem) [inline src/pokemon_storage_system.c]: `ITEM_NONE`

File: src/pokemon_storage_system.c [hunk 4]
```diff
@@ -4496,9 +4496,9 @@ static void sub_80CAEAC(void)
             sub_80D0D8C(CURSOR_AREA_IN_BOX, GetBoxCursorPosition());
     }
 
-    if (gUnknown_02039D12 != 0)
+    if (sMovingItemId != 0)
```
Comment (huderlem) [inline src/pokemon_storage_system.c]: `ITEM_NONE`

File: src/pokemon_storage_system.c [hunk 5]
```diff
@@ -367,7 +367,7 @@ enum
 {
     MODE_PARTY,
     MODE_BOX,
-    MODE_2,
+    MODE_MOVING,
```
Comment (huderlem) [inline src/pokemon_storage_system.c]: `MODE_MOVE` would be a slight improvement in my eyes.

File: src/pokemon_storage_system.c (hunk 1 above)
Comment (AsparagusEduardo) [reply src/pokemon_storage_system.c]: I based the name on from what appeared in `pokenav_conditions_2.c`. Should that also be changed?

**Pull request merged**